### ISSUE TITLE
Broken on windows after 0.14.1

### ISSUE THREAD

**Issue by ThibaultVlacich** [huang-julien/nitro-applicationinsights#121]
### Environment
Windows + Nuxt 3.14

### Reproduction
Add nuxt-applicationinsights to a nuxt project

Pin version of nitro-applicationinsights  to 0.14.0 -> No error
Pin to 0.14.1 -> Error

### Describe the bug
[nuxt] [request error] [unhandled] [500] Only URLs with a scheme in: file, data, and node are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'c:'

**Comment by huang-julien** [comment]
Hi :wave: I can't reproduce it on a bare nuxt project. Can you provide a lockfile + the package json please ?

**Comment by ThibaultVlacich** [comment]
I've upgraded nuxt-applicationinsights to 0.7.0, which upgraded nitro-applicationinsights to 0.14.2 and everything is working fine now :)
I'm closing

**Comment by fristys** [comment]
This is still an issue on Windows, running Node v22.11.0 /  npm v10.9.0 and `"nuxt-applicationinsights": "^0.7.0"`

**Comment by ThibaultVlacich** [comment]
Indeed, still an issue, but only in build mode, but not in dev mode anymore.

Here is a repro: https://github.com/ThibaultVlacich/nitro-appinsights-windows

```
npm run build
node .output/server/index.mjs
```

**Comment by huang-julien** [comment]
I love windows 🙃 ... i'm going to fix it ASAP after Nuxtnation

**Comment by huang-julien** [comment]
I'm going to release it as a minor (0.15) since I removed `#applicationinsights` alias. 

I'll plan 1.0 with applicationinsights 3 and the usage of opentelemetry
